atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/Event.java

```diff
@@ -41,5 +41,14 @@ static Event fromKeywordMap(Map<Keyword, ?> map) {
         return new ObjectMapper().convertValue(convertedMap, Event.class);
     }
 
+    static Map<Keyword, Object> toKeywordMap(Event event) {
+        Map<String, Object> stringToObject = new ObjectMapper().convertValue(event, Map.class);
+        Map<Keyword, Object> convertedMap = new HashMap<>();
+        EntryStream.of(stringToObject)
```

Comment:
nit: I think you can inline `convertedMap`, by doing `return EntryStream.of(stringToObject).mapKeys(Keyword::intern).toMap()` directly. I'm fine either way